sekoia_automation/module.py

```diff
@@ -384,13 +404,30 @@ def log_exception(self, exception: Exception, **kwargs):
                 scope.set_extra(key, value)
             sentry_sdk.capture_exception(exception)
 
-    @property
+    @cached_property
     def callback_url(self) -> str:
         if self._callback_url is None:
             self._callback_url = self.module.load_config(self.CALLBACK_URL_FILE_NAME)
-
         return self._callback_url
 
+    @cached_property
+    def logs_url(self) -> str:
+        if self._logs_url is None:
+            try:
+                self._logs_url = self.module.load_config(self.LOGS_URL_FILE_NAME)
+            except FileNotFoundError:
+                self._logs_url = self.callback_url.replace("/callback", "/logs")
+        return self._logs_url
+
+    @cached_property
+    def secrets_url(self) -> str:
+        if self._secrets_url is None:
+            try:
+                self._secrets_url = self.module.load_config(self.SECRETS_URL_FILE_NAME)
+            except FileNotFoundError:
+                self._secrets_url = self.callback_url.replace("/callback", "/secrets")
+        return self._secrets_url
```

Comment:
No need to use a private attribute for cached properties

```suggestion
    @cached_property
    def logs_url(self) -> str:
        try:
            return self.module.load_config(self.LOGS_URL_FILE_NAME)
        except FileNotFoundError:
            return = self.callback_url.replace("/callback", "/logs")

    @cached_property
    def secrets_url(self) -> str:
        try:
            return self.module.load_config(self.SECRETS_URL_FILE_NAME)
        except FileNotFoundError:
            return self.callback_url.replace("/callback", "/secrets")
```